airbyte-integrations/connectors/source-faker/metadata.yaml

```diff
@@ -32,6 +32,9 @@ data:
     - language:python
   releases:
     breakingChanges:
+      5.0.0:
+        message: "ID and products.year fields are changing to be integers instead of floats."
+        upgradeDeadline: "2023-08-31"
```

Comment:
cc @erohmensing - a wild breaking change has appeared

Reply:
I swear I'm not _trying_ to be a conn ops early adopter, it just keeps happening :p